console/src/views/Radio.js

```diff
@@ -0,0 +1,193 @@
+import Button from '@enact/agate/Button';
+import Divider from '@enact/agate/Divider';
+import {LabeledItemBase} from '@enact/agate/LabeledItem';
+import {Panel} from '@enact/agate/Panels';
+import ToggleButton from '@enact/agate/ToggleButton';
+import kind from '@enact/core/kind';
+import React, {Component} from 'react';
+
+import PresetItem from '../components/PresetItem';
+
+import css from './Radio.less';
+
+
+const RadioBase = kind({
+	name: 'Radio',
+
+	handlers: {
+		onFrequencyToggle: (ev, {changeFrequency}) => {
+			const frequency = ev.currentTarget.innerText.trim();
+			changeFrequency(frequency);
+		},
+		onPresetClick: (ev, {changeStation, presets}) => {
+			changeStation(presets[ev.presetIndex]);
+		},
+		onPresetDown: (ev, {changePreset}) => {
+			changePreset(ev.presetIndex);
+		},
+		onPresetHold: (ev, {currentStation, currentPreset, updatePresets}) => {
+			updatePresets(currentStation, currentPreset);
```

Comment:
Going along with my comments on `onTune`, we could consider emitting a `onUpdatePreset` with an object payload vs explicitly calling a method. This is "userspace" code so the framework-grade structure isn't really necessary but keeping the separation clean can often be useful and reduce later refactoring.

Same would apply to the other handlers here.